torchvision/ops/boxes.py

```diff
@@ -112,8 +113,16 @@ def clip_boxes_to_image(boxes, size):
     boxes_x = boxes[..., 0::2]
     boxes_y = boxes[..., 1::2]
     height, width = size
-    boxes_x = boxes_x.clamp(min=0, max=width)
-    boxes_y = boxes_y.clamp(min=0, max=height)
+
+    if torchvision._is_tracing():
+        boxes_x = torch.max(boxes_x, torch.tensor(0.))
+        boxes_x = torch.min(boxes_x, torch.tensor(width, dtype=torch.float32))
+        boxes_y = torch.max(boxes_y, torch.tensor(0.))
+        boxes_y = torch.min(boxes_y, torch.tensor(height, dtype=torch.float32))
+    else:
+        boxes_x = boxes_x.clamp(min=0, max=width)
```

Comment:
From your previous comment, 
> I believe clamp is traces with constants for min/max:
> Tensor clamp(const Tensor & self, c10::optional min, c10::optional max)
> 
> Would this cause a failure with branch:
> if torchvision._is_tracing():

Can we make it trace with tensors as well if tensors are passed?

Reply:
Here we have the same issue. The traced graph for clamp shows min/max as tensors even when a tensor is passed.
  %7 : int =  prim::Constant[value=2]()
  %8 : None = prim::Constant()
  %9 : Float(3, 4) = aten::clamp(%0, %7, %8) 
I haven't looked deeper.
The PR to address this: https://github.com/pytorch/pytorch/pull/32587
Maybe we can have this branch until the PR is merged.
I'm not sure if changing it to tensors would help much. Let me know what you think. Thanks.